### PR TITLE
i18n(es):  update `environment-variables.mdx`

### PR DESCRIPTION
**Pull request by Mardroide**
#### What kind of changes does this PR include?
- New or updated content

#### Reference
- [69c3ebd](https://github.com/withastro/docs/commit/69c3ebd748889429cf1f1b1a42e02bd2b0281cd7#diff-e5afe89538d79a61ba763693ce8702800a5489beac44d3c5be2317ee73924c0f)

### CODE REVIEW

**Review by dreyfus92 — APPROVED** [review]
LGTM ✅

File: src/content/docs/es/guides/environment-variables.mdx [hunk 1]
```diff
@@ -102,8 +102,7 @@ const data = fetch(`${import.meta.env.PUBLIC_POKEAPI}/pokemon/squirtle`);
...
-Astro verificará el entorno del servidor en busca de variables; si no existen, buscará en archivos .env.
+Cuando se utiliza SSR, se puede acceder a las variables de entorno en tiempo de ejecución en función del adaptador SSR que se esté utilizando. Con la mayoría de los adaptadores puede acceder a las variables de entorno con `process.env`, pero algunos adaptadores funcionan de forma diferente. Para el adaptador Deno, utilizará `Deno.env.get()`. Vea cómo [acceder al tiempo de ejecución de Cloudflare](/es/guides/integrations-guide/cloudflare/#cloudflareruntime) para manejar las variables de entorno cuando utilice el adaptador de Cloudflare. Astro comprobará primero el entorno del servidor en busca de variables, y si no existen, Astro las buscará en los archivos .env.
```
Comment (Waxer59) [inline src/content/docs/es/guides/environment-variables.mdx:105]:
```suggestion
Cuando se utiliza SSR, se puede acceder a las variables de entorno en tiempo de ejecución en función del adaptador SSR que se esté utilizando. Con la mayoría de los adaptadores puede acceder a las variables de entorno con `process.env`, pero algunos adaptadores funcionan de forma diferente. Para el adaptador Deno, utilizará `Deno.env.get()`. Mira cómo [acceder al runtime de Cloudflare](/es/guides/integrations-guide/cloudflare/#cloudflareruntime) para manejar las variables de entorno cuando utilices el adaptador de Cloudflare. Astro comprobará primero el entorno del servidor en busca de variables, y si no existen, Astro las buscará en los archivos .env.
```

**Review by Waxer59 — COMMENTED** [review]
Buena PR 🤠, te dejo una sugerencia

File: src/content/docs/es/guides/environment-variables.mdx [hunk 2]
```diff
@@ -102,8 +102,7 @@ const data = fetch(`${import.meta.env.PUBLIC_POKEAPI}/pokemon/squirtle`);
...
-Astro verificará el entorno del servidor en busca de variables; si no existen, buscará en archivos .env.
+Cuando se utiliza SSR, se puede acceder a las variables de entorno en tiempo de ejecución en función del adaptador SSR que se esté utilizando. Con la mayoría de los adaptadores puede acceder a las variables de entorno con `process.env`, pero algunos adaptadores funcionan de forma diferente. Para el adaptador Deno, utilizará `Deno.env.get()`. Mira cómo [acceder al runtime de Cloudflare](/es/guides/integrations-guide/cloudflare/#cloudflareruntime) para manejar las variables de entorno cuando utilices el adaptador de Cloudflare. Astro comprobará primero el entorno del servidor en busca de variables, y si no existen, Astro las buscará en los archivos .env.
```
Comment (Waxer59) [inline src/content/docs/es/guides/environment-variables.mdx:105]:
```suggestion
Cuando se utiliza SSR, se puede acceder a las variables de entorno en tiempo de ejecución en función del adaptador SSR que se esté utilizando. Con la mayoría de los adaptadores puedes acceder a las variables de entorno con `process.env`, pero algunos adaptadores funcionan de forma diferente. Para el adaptador Deno, utilizará `Deno.env.get()`. Mira cómo [acceder al runtime de Cloudflare](/es/guides/integrations-guide/cloudflare/#cloudflareruntime) para manejar las variables de entorno cuando utilices el adaptador de Cloudflare. Astro comprobará primero el entorno del servidor en busca de variables, y si no existen, Astro las buscará en los archivos .env.
```

**Review by Waxer59 — APPROVED** [review]
LGTM ✅

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Great work peeps 🚀

**Pull request merged**